nb_conda_kernels/nb_conda_kernels.py

```diff
@@ -27,10 +27,10 @@ def _conda_info(self):
 
         return conda_info
 
-    def _python_executable(self):
-        """Find the python executable for each env where jupyter is installed.
+    def _all_executable(self):
```

Comment:
Haven't tried this out yet, but it cries out for an even more modular approach: we've gotten over the "do it twice" hurdle, but i feel like a little class/registration method/multipledispatch/entry_point would make this extensible to cover other kernels.